purchases/index.js

```diff
@@ -506,7 +506,11 @@ function renderHtmlReceipt(purchase) {
     return mustache.render(receiptEmailLayout, purchase.data.viewModel);
 }
 
-function listPurchases(db, callback) {
+function listPurchases(db, options, callback) {
+    if(!callback){
+        callback = options;
+        options = {};
+    }
```

Comment:
Really nice commit, this one!!! Clean, smooth, easy!

If you wanted to include the first option in this commit, too, that would also be fine.